packages/build/CHANGELOG.md

```diff
@@ -2,3917 +2,4279 @@
 
 ### Dependencies
 
-* The following workspace dependencies were updated
-  * dependencies
-    * @netlify/config bumped from ^18.0.2 to ^18.1.0
+- The following workspace dependencies were updated
+  - dependencies
+    - @netlify/config bumped from ^18.0.2 to ^18.1.0
 
 ### Dependencies
 
-* The following workspace dependencies were updated
-  * dependencies
-    * @netlify/config bumped from ^18.1.0 to ^18.1.1
+- The following workspace dependencies were updated
+  - dependencies
+    - @netlify/config bumped from ^18.1.0 to ^18.1.1
 
 ### Dependencies
 
-* The following workspace dependencies were updated
-  * dependencies
-    * @netlify/config bumped from ^18.1.3 to ^18.1.4
+- The following workspace dependencies were updated
+  - dependencies
+    - @netlify/config bumped from ^18.1.3 to ^18.1.4
 
 ### Dependencies
 
-* The following workspace dependencies were updated
-  * dependencies
-    * @netlify/config bumped from ^18.2.0 to ^18.2.1
+- The following workspace dependencies were updated
+  - dependencies
+    - @netlify/config bumped from ^18.2.0 to ^18.2.1
```

Comment:
I know this file is auto-generated, but these 4 dependencies paragraphs should be removed at some point, not necessarily in this PR.